extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosImporter.java

```diff
@@ -292,12 +296,41 @@ long importPhotoBatch(
     return totalBytes;
   }
 
+  private void logMissingAlbumDetails(UUID jobId,
```

Comment:
(don't let this block your PR - merge and fix this later when you have time) looks like this is using different formatting; can you let a formatter run over your changes? (I _think_ this project seems to be using `google-java-formatter` - others can correct me)